SW801F20/P8
# Feature gps snapshot

**Features added**
This pr adds functionality for the following functionality for the map:
- Initially using the network or GPS to place yourself on the map, posts it to db
- Every 5 seconds the map is updated with the position of all the users, including yourself
- The distance to the other users is calculated
- The other users' yaw is used to display their orientation, this does however look pretty confusing with the markers currently used for other users

**How to review**
- Open the map
- Locate the other users
- Wait 5 seconds
- Tap the other users to see the distance to them

**Most important files**
- MapsViewModel
- MapsActivity

**This solves the following trello tasks:**
[https://trello.com/c/Ex4pR6Eb/218-update-map-with-db-locations](url)
[https://trello.com/c/dG2wWCXu/216-gps-snapshotting-retrieve-gps-location-of-user](url)
[https://trello.com/c/K7dW94Va/223-post-update-gps-snapshot-of-user-to-db](url)
[https://trello.com/c/imdwHRcp/224-get-current-users-location-from-db-since-dr-is-calculated-server-side](url)
[https://trello.com/c/SVqUXMMD/225-calculate-distance-to-other-markers-is-currently-hardcoded](url)
